src/PhpParser/Node/BetterNodeFinder.php

```diff
@@ -225,6 +225,7 @@ public function findFirstNonAnonymousClass(array $nodes): ?Node
 
     /**
      * @param Node|Node[] $nodes
+     * @param callable(Node $node): bool $filter
```

Comment:
@staabm this is somehow make invalid rectify, ref https://github.com/rectorphp/rector-src/pull/1649/commits/620d2b312559b02e0c5fd6bd8f3e1019cc1a2b7d

Reply:
Hmm it seems rector is mixing up the parameters